packages/config/config/devices/0x014f/zwn-bpc.json

```diff
@@ -11,7 +11,7 @@
 	],
 	"firmwareVersion": {
 		"min": "0.0",
-		"max": "5.9"
+		"max": "255.255"
```

Comment:
see my other comment regarding ZWN-BPC (with different manufacturer ID). I think 5.10+ is actually a different device.